dataactbroker/handlers/dashboard_handler.py

```diff
@@ -566,12 +566,12 @@ def active_submission_overview(submission, file, error_level):
     response['agency_name'] = agency.agency_name
     response['icon_name'] = agency.icon_name
 
-    # Deadline information, updates the default values of N/A only if it's a quarter format and the deadline exists
-    if submission.is_quarter_format:
-        deadline = sess.query(QuarterlyRevalidationThreshold.window_end).\
-            filter_by(year=submission.reporting_fiscal_year, quarter=submission.reporting_fiscal_period // 3).first()
+    # Deadline information, updates the default values of N/A only if it's not a test and the deadline exists
+    if not submission.test_submission:
```

Comment:
A thought went through my head of how this would affect the Active DABS Dashboard which we have the whole epic for. If I'm understanding right, this is just providing the details for a specific submission provided in the params. So the listing of submissions for the dashboard may still return only quarterly submissions and we're safe (dunno really)? Regardless I _think_ this is still fine. Just making sure we're on the same page on how broken the dashboard may get. My apologies for the word vomit.

Reply:
This does indeed only list the submission requested. This is also only active submissions so it's irrelevant because we already do monthly ones here.